test/e2e/scalability/dns.go

```diff
@@ -0,0 +1,91 @@
+/*
```

Comment:
Could you instead place this test under `test/e2e/network`?

Sig-scalability is a horizontal effort - scale testing for vertical features should ideally belong to the relevant sig (which in this case seems to be sig-networking).